src/wallet/wallet2.cpp

```diff
@@ -3776,12 +3779,16 @@ void wallet2::load(const std::string& wallet_, const epee::wipeable_string& pass
   boost::system::error_code e;
   bool exists = boost::filesystem::exists(m_keys_file, e);
   THROW_WALLET_EXCEPTION_IF(e || !exists, error::file_not_found, m_keys_file);
+  m_keys_file_locker.reset(new tools::file_locker(m_keys_file.c_str()));
+  THROW_WALLET_EXCEPTION_IF(!m_keys_file_locker->locked(), error::wallet_internal_error, "internal error: \"" + m_keys_file + "\" is opened by another wallet program");
 
+  m_keys_file_locker.reset();
```

Comment:
I realized this temporary unlocking was necessary due to Windows (otherwise the file couldn't be loaded).

Reply:
Maybe add a comment saying so ?

Reply:
ok